pkgs/by-name/ma/mattermost-desktop/package.nix

```
@@ -59,7 +65,7 @@ buildNpmPackage rec {
       $out/share/applications \
       $out/share/icons/hicolor/512x512/apps
 
-    readonly dist=release/linux-unpacked
+    readonly dist=${dist_path}
```

Maybe we could get away with a glob? So that we won’t need a case for riscv or something later?

```suggestion
    readonly dist=release/*-unpacked
```

---

The glob worked locally for me :+1: